composer.json

```diff
@@ -12,6 +12,7 @@
     "require": {
         "php": "^7.1.3",
         "symfony/process": "^4.2",
+        "symfony/console": "dev-master",
```

Comment:
You can't do that in the "prod" require. I suggest to change on .travis.yml:
https://github.com/nunomaduro/larastan/pull/306/files#diff-354f30a63fb0907d4ad57269548329e3

Reply:
It will also need to change the minimum-stability and maybe add an alias but the whole thing should done in Travis-CI only, and symfony/console will just be properly update when Symfony 4.4 will be released.